app/src/main/AndroidManifest.xml

```diff
@@ -1,57 +1,63 @@
-    -->
-    <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-
-    <application
-        android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
-        android:theme="@style/Theme.PolyBooks">
-        <meta-data android:name="com.google.android.gms.version" android:value="@integer/google_play_services_version"/>
-        <meta-data android:name="com.google.android.gms.vision.DEPENDENCIES" android:value="barcode"/>
-        <meta-data
-            android:name="preloaded_fonts"
-            android:resource="@array/preloaded_fonts" />
-
-        <activity
-            android:name=".ScanBarcodeActivity"
-            android:parentActivityName=".AddSaleActivity" />
-        <activity android:name=".FillSaleActivity"
-            android:parentActivityName=".AddSaleActivity"/>
-        <activity android:name=".AddSaleActivity"
-            android:parentActivityName=".MainActivity"/>
-
-        <activity android:name=".LoginActivity"
-            android:parentActivityName=".MainActivity"/>
-
-        <activity android:name=".UserProfileActivity"
-            android:parentActivityName=".MainActivity"/>
-
-        <activity android:name=".FilteringSalesActivity" />
-        <activity
-            android:name=".ListSalesActivity"
-            android:parentActivityName=".FilteringSalesActivity" />
-        <activity
-            android:name=".FilteringBooksActivity"
-            android:parentActivityName=".MainActivity"/>
-
-        <activity android:name=".MainActivity">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-        </activity>
-    </application>
-
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.github.polybooks">
+    <!--
+    Should we only allow for back-cameras? If yes, keep this line commented out
+    <uses-feature android:name="android.hardware.camera.any" />
+    -->
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.PolyBooks">
+        <activity android:name=".SaleInformationActivity"></activity>
```

Comment:
You need to specify a parent Activity so that the arrow to return to the previous activity is visible.